charts/dapr/crds/subscription.yaml

```diff
@@ -54,9 +54,9 @@ spec:
               deadLetterTopic:
                 type: string
               bulkSubscribe:
-                description: Represents bulk subscribe properies
```

Comment:
Interestingly, these descriptions are not in the objects that the CRDs are auto-generated from: https://github.com/dapr/dapr/blob/master/pkg/apis/subscriptions/v2alpha1/types.go